tests/playbooks/known_issues.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Set some global known_issues vars
+  set_fact:
+    skip_pvc: false
+    skip_vm: false
+    skip_dv_vm: false
+    skip_preset: false
+    skip_vmir: false
+    skip_e2e: false
+    current_playbook: ''
+
+    github_issues:
+    - name: pvc_module_242
+      url: https://api.github.com/repos/kubevirt/ansible-kubevirt-modules/issues/242
+
+- name: Fetch github issues
+  include_tasks: known_issues/github.yaml
+  with_items: "{{ github_issues }}"
+
+- name: "Known issue: ansible.git@devel breaks kubevirt_pvc #242"
+  set_fact:
+    skip_pvc: true
+    skip_e2e: true
+  when: (ansible_version.full == "2.9.0" or ansible_version.full == "2.8.1") and pvc_module_242.state == "open"
```

Comment:
Why to depend on specific z-stream? Do we want to update this number whenever new release is available?

Reply:
When a new release is available, we'll remove the whole test. We (will) test against:
- released (pip install ansible) – 2.8.1
- stable branch – 2.8.2 (to be released)
- devel branch – 2.9.0 (to be released)

If any of those have a known issue, then we need to skip certain tests, but *only* for affected branch/release. At present, the affected versions for this case are: released 2.8.1 and stable-2.8 branch. And I'm gonna put only those two versions for skipping, since I do want to run all of my tests on devel.